kubernetes/zenko/charts/burry/templates/cronjob.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "burry.fullname" . }}
+  labels:
+    app: {{ template "burry.name" . }}
+    chart: {{ template "burry.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+spec:
+  suspend: {{ .Values.cronjob.suspend }}
+  schedule: "{{ .Values.cronjob.schedule }}"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 3
+  successfulJobsHistoryLimit: 6
+  jobTemplate:
+    metadata:
+      labels:
+        app: {{ template "burry.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ template "burry.name" . }}
+            release: {{ .Release.Name }}
+{{- if .Values.pod.annotations }}
+          annotations:
+{{ toYaml .Values.pod.annotations | indent 12 }}
+{{- end }}
+        spec:
+          volumes:
+            - name: {{ template "burry.fullname" . }}-configmap
+              configMap:
+                name: {{ template "burry.fullname" . }}-config
+                items:
+                - key: .burryfest
+                  path: .burryfest
+            - name: burry-storage
+{{- if .Values.persistentVolume.enabled }}
+              persistentVolumeClaim:
+                claimName: {{ if .Values.persistentVolume.existingClaim }}{{ .Values.persistentVolume.existingClaim }}{{- else }}{{ template "burry.fullname" . }}{{- end }}
+{{- else }}
+              emptyDir: {}
+{{- end }}
+          restartPolicy: OnFailure
+          containers:
+            - name: {{ template "burry.fullname" . }}
```

Comment:
There's no need to template the full name for the container name.

Reply:
Agreed.